api/src/migrations/1619727566425-CreateEventEntity.ts

```diff
@@ -22,6 +25,9 @@ export class CreateEventEntity1619727566425 implements MigrationInterface {
                 id uuid NOT NULL DEFAULT uuid_generate_v4(),
                 previous character varying COLLATE pg_catalog."default" NOT NULL,
                 updated character varying COLLATE pg_catalog."default" NOT NULL,
+                field character varying,
+                subfield character varying,
+                language character varying,
                 created_at timestamp without time zone NOT NULL DEFAULT now(),
```

Comment:
oh sorry, I'm wrong. 
We need to add new migration file like, instead of adding here. (or we have to redo all migrations and run again in the POD. which will clean all the data)
Because this migration is already run.

![image](https://user-images.githubusercontent.com/46575401/117217554-6714c680-adb6-11eb-9e75-1cdc6bef90d0.png)

Reply:
something like in add the following in `package.json`, have to check typeorm doc for exactly command.
`migrate:revert: ./node_modules/.bin/ts-node ./node_modules/.bin/typeorm migration:revert --file $1`

then go to the POD terminal, running this `npm run migrate:revert xxxx.ts`

Reply:
ah good call, thanks!

Reply:
I will recommend add a revert migration scripts for better consistency, changing any remote db is always hi-risk job. 
And also add `IF EXISTS` or `IF NOT EXISTS` clause in your sql command.